charms/katib-controller/requirements-integration.in

```diff
@@ -0,0 +1,4 @@
+juju==2.9.44
+lightkube
+pytest-operator
+-r requirements.txt
```

Comment:
we've recently concluded that requirements.txt is not needed here since it is packed with the charm.